db/seeds.rb

```diff
@@ -1,5 +1,18 @@
-unless Customer.exists?
-  125.times do
-    Customer.create! name: Faker::Name.name, email: Faker::Internet.email, company: Faker::Company.name, age: rand(18..70), status: ["Active", "Inactive", "Pending", "Suspended"].sample
+50.times do
```

Comment:
I don't want you to change `seeds.rb` - please revert.

Instead, what I suggest is that you move your suggested code to a rake task that can be run. We can add a note in the docs about it so people can run their own benchmarks.